msprime/trees.py

```diff
@@ -2320,7 +2320,6 @@ def node(self, u):
     def time(self, sample):
         return self.get_time(sample)
 
-    # TODO refactor these methods to take node arguments not samples.
     def get_time(self, sample):
```

Comment:
We should change the parameter here to ``u`` (also for the ``time`` method above), and change the documentation so that sample -> node.